src/main/java/com/marklogic/appdeployer/command/hosts/AssignHostsToGroupsCommand.java

```diff
@@ -0,0 +1,60 @@
+package com.marklogic.appdeployer.command.hosts;
+
+import java.util.Map;
+
+import com.marklogic.appdeployer.command.AbstractUndoableCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.mgmt.admin.AdminManager;
+import com.marklogic.mgmt.resource.appservers.ServerManager;
+import com.marklogic.mgmt.resource.hosts.HostManager;
+
+public class AssignHostsToGroupsCommand extends AbstractUndoableCommand {
+
+	private static final String MANAGE_SERVER_NAME = "Manage";
+	private static final String MANAGE_SERVER_JSON_TEMPLATE = "{\"server-name\":\"%s\", \"group-name\":\"%s\", \"server-type\":\"http\", \"enabled\":true, \"root\":\"Apps/\", \"authentication\":\"digest\", \"port\":8002, \"webDAV\":false, \"execute\":true, \"display-last-login\":false, \"address\":\"0.0.0.0\", \"backlog\":512, \"threads\":32, \"request-timeout\":30, \"keep-alive-timeout\":5, \"session-timeout\":3600, \"max-time-limit\":3600, \"default-time-limit\":600, \"max-inference-size\":500, \"default-inference-size\":100, \"static-expires\":3600, \"pre-commit-trigger-depth\":1000, \"pre-commit-trigger-limit\":10000, \"collation\":\"http://marklogic.com/collation/\", \"internal-security\":true, \"concurrent-request-limit\":0, \"compute-content-length\":true, \"log-errors\":false, \"debug-allow\":true, \"profile-allow\":true, \"default-xquery-version\":\"1.0-ml\", \"multi-version-concurrency-control\":\"contemporaneous\", \"distribute-timestamps\":\"fast\", \"output-sgml-character-entities\":\"none\", \"output-encoding\":\"UTF-8\", \"output-method\":\"default\", \"output-byte-order-mark\":\"default\", \"output-cdata-section-namespace-uri\":\"\", \"output-cdata-section-localname\":null, \"output-doctype-public\":\"\", \"output-doctype-system\":\"\", \"output-escape-uri-attributes\":\"default\", \"output-include-content-type\":\"default\", \"output-indent\":\"default\", \"output-indent-untyped\":\"default\", \"output-indent-tabs\":\"default\", \"output-media-type\":\"\", \"output-normalization-form\":\"none\", \"output-omit-xml-declaration\":\"default\", \"output-standalone\":\"omit\", \"output-undeclare-prefixes\":\"default\", \"output-version\":\"\", \"output-include-default-attributes\":\"default\", \"default-error-format\":\"compatible\", \"error-handler\":\"manage/error-handler.xqy\", \"url-rewriter\":\"manage/rewriter.xqy\", \"rewrite-resolves-globally\":false, \"ssl-allow-sslv3\":true, \"ssl-allow-tls\":true, \"ssl-disable-sslv3\":false, \"ssl-disable-tlsv1\":false, \"ssl-disable-tlsv1-1\":false, \"ssl-disable-tlsv1-2\":false, \"ssl-hostname\":\"\", \"ssl-ciphers\":\"ALL:!LOW:@STRENGTH\", \"ssl-require-client-certificate\":true, \"content-database\":\"App-Services\", \"default-user\":\"nobody\", \"privilege\":\"http://marklogic.com/xdmp/privileges/manage\"}";
```

Comment:
Trim this down to only the fields that matter - most of these are default values that don't need to be specified. I'm guessing you need - name, root, port, group, database, privilege, error-handler, and url-rewriter - that should be it.